frontend/pages/beta/model/[modelId]/access-request/[accessRequestId].tsx

```diff
@@ -0,0 +1,74 @@
+import ArrowBack from '@mui/icons-material/ArrowBack'
+import { Button, Card, Container, Divider, Stack, Typography } from '@mui/material'
+import { useGetAccessRequest } from 'actions/accessRequest'
+import { useGetReviewRequestsForModel } from 'actions/review'
+import { useRouter } from 'next/router'
+import EmptyBlob from 'src/common/EmptyBlob'
+import Loading from 'src/common/Loading'
+import ReviewComments from 'src/common/ReviewComments'
+import MultipleErrorWrapper from 'src/errors/MultipleErrorWrapper'
+import Link from 'src/Link'
+import AccessRequestFormEditPage from 'src/model/beta/accessRequests/AccessRequestFormEditPage'
+import ReviewBanner from 'src/model/beta/reviews/ReviewBanner'
+import Wrapper from 'src/Wrapper.beta'
+
+export default function AccessRequest() {
+  const router = useRouter()
+
+  const { modelId, accessRequestId }: { modelId?: string; accessRequestId?: string } = router.query
+
+  const { accessRequest, isAccessRequestLoading, isAccessRequestError } = useGetAccessRequest(modelId, accessRequestId)
+  const {
+    reviews: activeReviews,
+    isReviewsLoading: isActiveReviewsLoading,
+    isReviewsError: isActiveReviewsError,
+  } = useGetReviewRequestsForModel({
+    modelId: accessRequest?.modelId,
+    accessRequestId: accessRequest?.id,
+    isActive: true,
+  })
+
+  const error = MultipleErrorWrapper('Unable to load access request', {
+    isAccessRequestError,
+    isActiveReviewsError,
+  })
+  if (error) return error
+
+  return (
+    <Wrapper
+      title={accessRequest ? (accessRequest.metadata.overview.name as string) : 'Loading...'}
+      page='access-request'
+      fullWidth
+    >
+      <Container maxWidth='md'>
+        <Card sx={{ p: 4, m: 'auto', mb: 2 }}>
+          {isAccessRequestLoading && isActiveReviewsLoading && <Loading />}
+          {!accessRequest && !isAccessRequestLoading && (
+            <EmptyBlob text={`Oh no, it looks like access request ${accessRequest} doesn't exist!`} />
```

Comment:
Is this useful? If we can't find the accessRequest the API should've returned a 404. So we'll be showing an error message instead of this anyway. I think this can probs be removed.